### DISCUSSION TITLE
Azure Authentication in Global Configuration Options

### DISCUSSION THREAD

**Discussion by erich-hs** [duckdb/duckdb#11933] [General]
Hi everyone,
I'm working with an Azure Data Lake Storage and authenticating using the Azure extension, but I see that there are no [global configuration options](https://duckdb.org/docs/configuration/overview.html) (similar to the ones for AWS S3 protocol) to support it.

Specifically, I'm looking at ways to set my DuckDB authentication settings to the ADLS within a dbt profiles.yml file.

Is an S3 proxy the only option, or are there plans to have it in the global configs in the near future?

**Comment by Tishj** [comment]
We use this for S3 related settings:
<https://duckdb.org/docs/configuration/secrets_manager>
